packages/bot-web-ui/src/components/dashboard/quick-strategy/quick-strategy-components/duration-fields.tsx

```diff
@@ -26,8 +26,9 @@ const DurationFields = ({
     handleChange,
     onChangeInputValue,
     setCurrentFocus,
-}: TDurationFields) =>
-    id === 'duration-unit' ? (
+}: TDurationFields) => {
+    const is_mobile = isMobile();
+    return id === 'duration-unit' ? (
```

Comment:
```suggestion
}: TDurationFields) => id === 'duration-unit' ? (
```

Reply:
@prince-deriv we need this line of the code :  `const is_mobile = isMobile();` because we use is_mobile in current component a few times